1039-wizard-design-guidance.md

```diff
@@ -0,0 +1,31 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-17
+updated: 2024-10-21
+---
+
+# Wizard Component
+
+A [Wizard](https://clarity.design/documentation/wizard) is a component that guides the user through a set of predefined steps. Wizards can help users focus on only the input needed for each step and decrease the chance of errors.
+
+## Guidance
+
+### Wizard Usage
+
+- **Must** use a wizard to help users walk through a defined step-by-step process.
```

Comment:
```suggestion
- **May** use a wizard to help users walk through a defined step-by-step process.
```

This is a "may" because the stepper is an option as well.